server/routes/oodler.js

```diff
@@ -22,15 +22,26 @@ function get(request, reply) {
 }
 
 function create(request, reply) {
-  return Oodler({
-    firstName: request.payload.firstName,
-    lastName: request.payload.lastName,
-    email: request.payload.email,
-    office: request.payload.office
-    })
-    .save()
-    .then(result => {
-      reply(result).code(201);
+  return Oodler
+    .filter({ email: request.payload.email})
+    .run()
+    .then(exists => {
+      if(exists) {
+        reply('User already registered with this email').code(400);
+      }
+
+    return Oodler({
+      firstName: request.payload.firstName,
+      lastName: request.payload.lastName,
+      email: request.payload.email,
+      password: request.payload.password,
```

Comment:
You're right. I had in mind register approach which we don't have... 
My bad :dizzy_face: